README.md

```diff
@@ -0,0 +1,60 @@
+# flup-py3 for Python 3.4+
+
+[flup-py3](https://hg.saddi.com/flup-py3.0/) fork and updated form Python 3.4+.
```

Comment:
s/form/for/

Reply:
Typo fixed in the new commit. Please check.